wrangler/src/txFunctions/list.js

```diff
@@ -0,0 +1,26 @@
+import { response } from 'cfw-easy-utils'
+
+export default async ({ request, env }) => {
+  const { TX_FUNCTIONS } = env
+
+  const url = new URL(request.url)
+  const query = Object.fromEntries(url.searchParams.entries())
+  const { limit, cursor: c } = query
+
+  let { list_complete, cursor, keys } = await TX_FUNCTIONS.list({ limit, cursor: c })
```

Comment:
Ha, Cloudflare does make some stuff easy, huh. Nice!

Reply:
Will this return compacted code, too? 

If it does, I think that might be a bit much to serve over an api request mostly for contract keys / hashes.

Reply:
It doesn't. `.list` just returns keys and metadata. You gave to call `.get` to get the actual values.